src/main/java/com/google/devtools/build/lib/profiler/Profiler.java

```diff
@@ -932,6 +941,12 @@ private void writeTask(JsonWriter writer, TaskData data) throws IOException {
       if (includePrimaryOutput && data instanceof ActionTaskData) {
         writer.name("out").value(((ActionTaskData) data).primaryOutputPath);
       }
+      if (includeTargetLabel && data instanceof ActionTaskData) {
+        writer.name("args");
```

Comment:
I assume you placed this in args to have it visible in `chrome://tracing`?

cc @joeleba we may want to consider the same for the primary output

Reply:
Yes, this makes it visible in Chrome. I see primary output to be in args as well, but may be too late to change it.
Trace file has pretty strict format about JSON fields allowed in it, however `chrome://tracing` parser is not very strict so it just skips unknown fields.
I guess it is better to stick with the format just for the sake of consistency if not visibility.

Reply:
My only concern would be file size. But since it is guarded behind a flag and after gzipping the difference is probably negligible your reasoning sounds valid.